benchmarks/java/src/main/java/com/google/protobuf/ProtoBenchCaliper.java

```diff
@@ -95,7 +98,7 @@ Message getDefaultInstance() {
   private List<ByteArrayInputStream> inputStreamList;
   private List<ByteString> inputStringList;
   private List<Message> sampleMessageList;
-  private int counter;
+  private long counter, mCounter;
```

Comment:
What is mCounter? Can you give them better names?

Reply:
This is used for get rid of the warmup runs which shouldn't be counted.

There're only 3 kinds of runs for caliper benchmark. 
One is dry run, usually just run for once and not in subprocess. Then follows warmup run,  which is constrained by running time, which is run in subprocess. Then are measurements, which follows up with warmup, and are constrained by number of measurements and one mearsurements running time.

Rewriten the counter to use an array to restore all the last (number of measurements + 1) counters.